.gitignore

```diff
@@ -8,6 +8,7 @@ pnpm-debug.log*
 lerna-debug.log*
 
 node_modules
+dist/*
```

Comment:
There is a copy and paste version in the `dist` folder, probably should leave that intact

Reply:
Fixed.